tests/data_context/cloud_data_context/test_datasource_crud.py

```diff
@@ -250,14 +236,15 @@ def json(self):
         ),
     ],
 )
-def test_data_context_in_cloud_mode_add_datasource(
+def test_cloud_data_context_add_datasource(
```

Comment:
Namespace collision with other test within this file

Reply:
Yes! Thank you - fixed in https://github.com/great-expectations/great_expectations/pull/5640 as well.

Reply:
Would it be useful to organize the cloud-data-context vs base-data-context in separate test classes?
https://docs.pytest.org/en/7.1.x/getting-started.html?highlight=testclass#group-multiple-tests-in-a-class

Then you can avoid the namespace collision and keep shorter test names.
Also makes it easier to maintain the appropriate pytest markers by simply marking the different test classes, instead of having to mark every test.

There are also some opportunities to declare shared parameters + fixtures in a single place. Although that might not be too useful in this particular case. 😄 
https://stackoverflow.com/a/62176555/6304433

Reply:
Ooh might be worth looking into - I'd vote to defer test refactoring to a subsequent PR/ticket just to get this work across the line but I'm happy to discuss!

Reply:
I agree we can do a better job of organizing the CRUD tests, but I'd vote for a separate PR to separate the diffs from moving things around.